docs/fundamentals/executor/executor-built-in-features.md

```diff
@@ -232,4 +232,33 @@ with (Flow().
         add(uses={'jtype': 'MyExecutor', 'metas': {'name': 'my-executor-1'}}).
         add(uses={'jtype': 'MyExecutor', 'metas': {'name': 'my-executor-2'}})) as f:
     f.post(on='/endpoint', inputs=DocumentArray([]), parameters={'param': 5, 'my-executor-1': {'param': 10}})
-```
\ No newline at end of file
+```
+
+
+````{admonition}  Note
+:class: caution
+
+As `parameters` does not have a fixed schema, it is declared with type `google.protobuf.Struct` in the `RequestProto`
+protobuf declaration. However, `google.protobuf.Struct` follows the JSON specification and does not 
+differentiate `int` from `float`. **So, data of type `int` in `parameters` will be casted to `float` when request is
+sent to executor.**
+
+As a result, users need be explicit and cast the data to the expected type as follows.
+
+```{code-block} python
+---
+emphasize-lines: 6
+---
+
+class MyExecutor(Executor):
+    animals = ['cat', 'dog', 'turtle']
+    @request
+    def foo(self, docs, parameters: dict, **kwargs):
+        # need to cast to int since list indices must be integers not float
+        index = int(parameters.get('index', 0))
+        assert self.animals[index] == 'dog'
+
+with Flow().add(uses=Selector) as f:
```

Comment:
```suggestion
with Flow().add(uses=MyExecutor) as f:
```

Reply:
Fixed [here](https://github.com/jina-ai/jina/pull/3349/commits/1edd61794053985266a3da7258ec215f96e28458), thank you.